kudos-webapps/src/main/webapp/vue-app/kudos/components/KudosApp.vue

```diff
@@ -27,11 +27,9 @@
           ref="activityKudosForm"
           class="flex mx-4">
           <div class="d-flex flex-column flex-grow-1">
-            <div class="d-flex flex-row pt-5 align-center">
-              <span class="text-header-title text-no-wrap">{{ $t('exoplatform.kudos.content.to') }}</span>
-              <div
-                v-if="isLinkedKudos"
-                class="d-flex flex-row pl-4 mb-2 text-truncate kudosReceiverAttendeeItem">
+            <div v-if="composerDisplay">
```

Comment:
We may need a dedicated component to define and inject into composer instead of reusing the existing one.

```suggestion
            <div>
```

Reply:
done!